docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+# Version 2 DICOM Conformance Statement
+
+> API version 2 is in **Preview** and should be used only for testing.
```

Comment:
Does this mean that that version is `/v2.0-prerelease`?

Reply:
currently it is /v2/ behind a Feature flag. They need to turn on a config setting to use it.